docs/references/glossary.md

```diff
@@ -139,11 +155,13 @@ A write stream of [documents](#documents).
 
 Someone who develops one or more plugins that enable Backstage to interoperate with another software system. A [user role](#user-role).
 
-## JWT
+## JSON Web Token
 
-JSON Web Token.
+A popular, compact, and self-contained open standard for securely transmitting information between parties as a JSON object. It is commonly used to verify a user's identity [authentication](#authentication) and permissions [authorization](#authorization). Each JWT consists of a header, payload, and digital signature separated by dots. While often encrypted, JWTs are always signed for integrity. This standard is a key component of [OpenID Connect](#openid-connect). For more details, see [the Wikipedia article](https://en.wikipedia.org/wiki/JSON_Web_Token).
 
-A popular JSON based token format that is commonly encrypted and/or signed, see [the Wikipedia article](https://en.wikipedia.org/wiki/JSON_Web_Token) for more details.
+## JWT
+
+[JSON Web Token](#json-web-token).
```

Comment:
I think we can merge these two perhaps? JSON Web Token (JWT) or AKA JWT? Something like this? 💭